mm2src/coins/eth/v2_activation.rs

```diff
@@ -215,11 +271,85 @@ impl EthCoin {
             logs_block_range: self.logs_block_range,
             nonce_lock: self.nonce_lock.clone(),
             erc20_tokens_infos: Default::default(),
+            nfts_infos: Default::default(),
             abortable_system,
         };
 
         Ok(EthCoin(Arc::new(token)))
     }
+
+    /// Creates a global NFT instance from a platform coin, such as Ethereum.
+    ///
+    /// This method initializes a new `EthCoin` instance specifically for handling global
+    /// non-fungible token based on a given platform coin. It sets up necessary configurations,
+    /// including required confirmations, web3 instances, and NFT information fetched from a provided URL.
+    ///
+    /// # Arguments
+    ///
+    /// * `ctx`: Shared application context.
+    /// * `chain`: Blockchain to which the NFTs belong.
+    /// * `conf`: Configuration parameters for the coin.
+    /// * `activation_request`: Activation request containing specific requirements.
+    /// * `url`: URL to fetch NFT data from.
```

Comment:
I don't think we should document the whole function signature. See https://github.com/KomodoPlatform/komodo-defi-framework/pull/2014#discussion_r1487411900

Reply:
Agree, done